frontend/controller/actions/identity.js

```diff
@@ -391,6 +407,9 @@ export default (sbp('sbp/selectors/register', {
         }
       }
 
+      // NOTE: update chatRoomLogs to the latest one
+      await sbp('gi.actions/identity/resetChatRoomLogs')
+
```

Comment:
> I think a better place for this would be after all the contracts are synced. So perhaps near line 468

I don't think so, because while syncing the chatroom contracts, it could be necessary to update `unreadMessages` (previous `chatRoomLogs`). So we should load the unreadMessages from the KV store as soon as the identity contract is synced.

> I would wrap this in its own try/catch block so that in case it fails it doesn't break the logging in process

Nice catch. I think we rather add try/catch in `fetchChatRoomUnreadMessages` since it's used in other places too. And this way will update the `loadChatRoomUnreadMessages` not to break.